katago_server/distributed_efforts/services/rating_network_pairer.py

```diff
@@ -93,7 +95,9 @@ def _choose_opponent(self, reference_network):
                 return None
             return np.random.choice(nearby_networks)
 
-        win_probability = np.asarray([1 / (1 + exp(opp_net.log_gamma - ref_net_log_gamma)) for opp_net in nearby_networks])
+        win_probability = np.asarray([
+            1 / (1 + exp((opp_net.log_gamma - ref_net_log_gamma) / self.current_run.rating_game_entropy_scale)) for opp_net in nearby_networks
```

Comment:
What does signify `self.current_run.rating_game_entropy_scale`

Reply:
If I understand right it is to vary the range of elo it could be matched on.

Reply:
Yes, it's just to be able to control the range of rating game opponents, if we need to.